portfolio/src/main/webapp/index.html

```diff
@@ -10,7 +10,32 @@
     <div id="content">
       <h1>My Portfolio - Jiang Yuxin</h1>
       <p>Hi everyone, I’m Yuxin. I’m a second year Computer Science student in NUS. In my free time, I love traveling, exploring food from different cultures and cooking. I also love watching TV series and movies.
-         My favourite sci-fic series is Black Mirror!  I am really excited to join Google as an intern this year and I am looking forward to getting to know you!</p>
+         My favourite sci-fic series is Black Mirror! I am really excited to join Google as an intern this year and I am looking forward to getting to know you!</p>
```

Comment:
Consider wrapping the longer lines. There is no column limit recommendation for HTML, but I think it'll improve the readability.
ref - https://google.github.io/styleguide/htmlcssguide.html#HTML_Line-Wrapping